flask/cli.py

```diff
@@ -874,6 +874,50 @@ def routes_command(sort, all_methods):
 ))
 
 
+def run_from_app_run(app, host=None, port=None, debug=None):
```

Comment:
This is now a third place, in addition to `app.run` and `run_command`, where a different subset of args is supported.

Reply:
Yes. It's unfortunate, but I see it as a way to converge the current two ways to start an app into a unified solution, because eventually the arguments in `app.run` could be deprecated in favor of CLI arguments.